Lib/sqlite3/deprecated.py

```diff
@@ -0,0 +1,10 @@
+def __getattr__(name):
+    if name == "OptimizedUnicode":
+        import warnings
+        msg = ("""
+            OptimizedUnicode is obsolete. You can safely remove it from your
+            code, as it defaults to 'str' anyway.
+        """)
+        warnings.warn(msg, DeprecationWarning)
```

Comment:
Add `stacklevel=2`.

You could also check the file name of the emitted warning in tests.

Reply:
Of course, thanks! Fixed in a9e959f

Reply:
Could've used `os.path.basename(__file__)` iso. `factory.py` in the unit test, but this test is bound to be short-lived anyway.

Reply:
Maybe just compare filename with `__file__`?

In any case, as you said, this is a short-lived test, so it does not matter.